### PR TITLE
Allow overriding CRS allowed HTTP methods per vhost

### PR DESCRIPTION
**Pull request by Vincevrp**

### CODE REVIEW

**Comment by Vincevrp** [comment]
@david22swan Rebased. Should be good to go now.

File: manifests/vhost.pp [hunk 1]
```diff
@@ -1913,6 +1916,7 @@
   Optional[String] $modsec_body_limit                                                 = undef,
   Optional[Integer[1, default]] $modsec_inbound_anomaly_threshold                     = undef,
   Optional[Integer[1, default]] $modsec_outbound_anomaly_threshold                    = undef,
+  Optional[String] $modsec_allowed_methods                                            = undef,
```
Comment (david22swan) [inline manifests/vhost.pp:1919]: @Vincevrp
Could you give me your reasoning on why this is a String and not and Array?

Comment (Vincevrp) [reply manifests/vhost.pp:1919]: @david22swan To be consistent with https://github.com/puppetlabs/puppetlabs-apache/blob/main/manifests/mod/security.pp#L124

**Review by david22swan — APPROVED** [review]
With that clarification on the type...

LGTM
Thank's for all the work you've put in his week :)

**Comment by david22swan** [comment]
Plan is for a release to go out today so you should have your changes availible pretty quick

**Comment by Vincevrp** [comment]
@david22swan Thanks for the quick feedback and merges. More contributions to come :)

**Pull request merged**